addons/web_editor/static/lib/odoo-editor/test/spec/list.test.js

```diff
@@ -3018,20 +3018,20 @@ describe('List', () => {
                             await testEditor(BasicEditor, {
                                 contentBefore: '<ol><li>abc</li><li>[]def</li></ol>',
                                 stepFunction: deleteBackward,
-                                contentAfter: '<ol><li>abc[]def</li></ol>',
+                                contentAfter: '<ol><li>abc</li></ol><p>[]def</p>',
                             });
                             // With another list item after.
                             await testEditor(BasicEditor, {
                                 contentBefore: '<ol><li>abc</li><li>[]def</li><li>ghi</li></ol>',
                                 stepFunction: deleteBackward,
-                                contentAfter: '<ol><li>abc[]def</li><li>ghi</li></ol>',
+                                contentAfter: '<ol><li>abc</li></ol><p>[]def</p><ol><li>ghi</li></ol>',
                             });
                             // Where the list item to merge into is empty, with an
                             // empty list item before.
                             await testEditor(BasicEditor, {
                                 contentBefore: '<ol><li><br></li><li><br></li><li>[]abc</li></ol>',
                                 stepFunction: deleteBackward,
-                                contentAfter: '<ol><li><br></li><li>[]abc</li></ol>',
+                                contentAfter: '<ol><li><br></li><li><br></li></ol><p>[]abc</p>',
```

Comment:
@Zinston Though it kind of defeats the purpose of this test here since we don't have the "removal of the BR that was there for display-purposes only" anymore.

Reply:
Yeah well same comment :-P
I also seem to remember specifically that FP preferred the behavior from before (he was quite specific with his expectations on these things and spent a while playing with lists with JW, that was one of his pet peeves).